backend/services/location.service.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable func-names */
+const { Location } = require('../models/dictionaries/location.model');
+const DatabaseError = require('../errors/database.error');
+
+const LocationService = {};
+
+function validateLocations(locationsarray){
```

Comment:
in all checks please throw [ValidationError](https://github.com/hackforla/VRMS/blob/development/backend/errors/validation.error.js) that contains specific message

Reply:
all checks now throwing validation error